__tests__/app.test.js

```diff
@@ -41,3 +41,49 @@ describe("GET/api/topics", () => {
       });
   });
 });
+
+describe("GET/api/articles", () => {
+  it("Responds with an error if path is not found", () => {
+    return request(app)
+      .get("/anything")
```

Comment:
a seperate test is needed to test that the endpoint of /api/anything returns a 404